doc/releases/changelog-dev.md

```diff
@@ -294,5 +298,5 @@ Vincent Michaud-Rioux,
 Anurav Modak,
 Mudit Pandey,
 Erik Schultheis,
-nate stemen,
-David Wierichs,
\ No newline at end of file
+Nate Stemen,
```

Comment:
I believe this was on purpose, nate spells his name in lower case, see https://github.com/natestemen, and https://unitary.fund/ (search for nate's name on this page)

Reply:
Oh alright, will revert!